src/status_im/contexts/shell/qr_reader/sheets/scanned_address.cljs

```diff
@@ -0,0 +1,26 @@
+(ns status-im.contexts.shell.qr-reader.sheets.scanned-address
```

Comment:
nit: we should discrimate somehow what this is related to.
status-im.contexts.shell.qr-reader.sheets.wallet-scanned-address
or 
status-im.contexts.shell.qr-reader.sheets.scanned-wallet-address

Reply:
Done